site/assets/js/tac.js

```diff
@@ -11,10 +11,12 @@
 
     tac.querySelectorAll('.tarteaucitronAllow').forEach(function (button) {
       button.classList.add('btn', 'btn-sm', 'btn-success', 'my-2', 'my-lg-0')
+      button.innerHTML = tarteaucitron.lang.acceptAll
```

Comment:
Is the key to use `acceptAll` ("OK, tout accepter") or `allowAll` ("Autoriser tous les cookies") ?
In the source code of tarteaucitron it is the `allowAll` key which seems to be used.

Reply:
Good catch! It is indeed `allowAll`.